feat(docs): view terminology around stream metadata (#778)
trufnetwork/node: TERMINOLOGY.md

- TN GOVERNANCE: The entity or group responsible for approving streams and managing the System Contract.
- SAFE READ: A query made through the System Contract that only returns data from official streams.
- UNSAFE READ: A query made through the System Contract that can return data from any stream, official or unofficial.

> - STREAM DEPLOYER: Entity (defined by a wallet address) that deploys Data Contracts (Primitive and Composed) on the TN.

Reviewer: ```suggestion
- STREAM DEPLOYER: A deployer (wallet address) of a specific stream.
```

> - STREAM OWNER: A wallet address that owns a Data Contract (Primitive or Composed). The owner does not necessarily deploy the stream but can create Composed Stream using Data Contracts deployed by other wallets.

Reviewer: ```suggestion
- STREAM OWNER: A wallet address that owns a stream contract. The Stream Owner is not necessarily the Stream Deployer but can create a Composed Stream using contracts deployed by other wallets.
```
